util/git_pre-commit.in

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright (c) 2017, Juan Hernando <juan.hernando@epfl.ch>
+
+GIT=${GIT_EXECUTABLE}
+CLANG_FORMAT=${CLANG_FORMAT}
```

Comment:
this will hardcode a path at configure time, not a good idea because this file will not be overwritten once it has been created. Just use the executable from the path?